test/controllers/api/v2/base_controller_subclass_test.rb

```diff
@@ -124,9 +124,9 @@ def setup
   test "should have server error message" do
     get :new
     assert_response 500
-    msg = "Internal Server Error: the server was unable to finish the request. "
-    msg << "This may be caused by unavailability of some required service, incorrect API call or a server-side bug. "
-    msg << "There may be more information in the server's logs."
+    msg = "Internal Server Error: the server was unable to finish the request. " + \
+      "This may be caused by unavailability of some required service, incorrect API call or a server-side bug. " + \
+      "There may be more information in the server's logs."
```

Comment:
```suggestion
    msg = "Internal Server Error: the server was unable to finish the request. " \
      "This may be caused by unavailability of some required service, incorrect API call or a server-side bug. " \
      "There may be more information in the server's logs."
```
Since this is just a long string divided to smaller strings, we can use plain `\` here to concatenate them.